en/00_Getting_Started/00_Server_Requirements.md

```diff
@@ -22,7 +22,9 @@ Use [phpinfo()](https://php.net/manual/en/function.phpinfo.php) to inspect your
 
 Silverstripe CMS tracks the official [PHP release support timeline](https://www.php.net/supported-versions.php). When a PHP version reaches end-of-life, Silverstripe CMS drops support for it in the next minor release.
 
-You also need to install [Composer 2](https://getcomposer.org/).
+You need to install [Composer 2](https://getcomposer.org/).
+
+To increase performance it's recommended that an [opcode](https://en.wikipedia.org/wiki/Opcode) cache such as [OPcache](https://www.php.net/manual/en/book.opcache.php) is used.
```

Comment:
Moved from index.md

opcache is talked about in a little more detail in 08_Other_Optimisaitons.md

Reply:
Updated